Code/Engine/Foundation/Strings/StringView.h

```diff
@@ -2,6 +2,7 @@
 
 #include <Foundation/Strings/Implementation/StringIterator.h>
 
+#include <string_view>
```

Comment:
The advantage of ez so far was, that we used almost no stl, and thus we were not affected form any of the STL code bloat. If we now start including things like <string_view> in a header file as central as StringView.h, we suddenly get all the STL code bloat. If we absolutely need this, it should be behind a compile flag, so it can be turned off, if you don't need it.

Reply:
My main fear is, that this is undoing most of my past efforts to make ez compile quickly.

Reply:
We already include <string> in those low level files. <string_view> should be relatively unproblematic. A compile flag is in practice not maintainable.

Reply:
No, unfortunately not. As soon as you switch C++20 on, **everything** from STL include `<ranges>` and bloats your compile times.

Reply:
Can't we make this a separate file and use the `MakeX` pattern for interop? Like `ezSTL::MakeString`, `ezSTL::MakeStlString`, `ezSTL::MakeStringView`, `ezSTL::MakeStlStringView` etc. There shouldn't be a need for putting this inside the class outside of implicent conversion convenience?